pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -166,6 +166,8 @@ void *initObjToJSON(void)
         cls_index = (PyTypeObject *)PyObject_GetAttrString(mod_pandas, "Index");
         cls_series =
             (PyTypeObject *)PyObject_GetAttrString(mod_pandas, "Series");
+        cls_timestamp = PyObject_GetAttrString(mod_pandas, "Timestamp");
+	cls_timedelta = PyObject_GetAttrString(mod_pandas, "Timedelta");
```

Comment:
Is this indentation off?

Reply:
I think just matching the block but generally indentation is off in the C files. We don’t have any linting here so I think a good follow up generally for extension modules. 

We have cpplint but I don’t think that really covers C code. gnu indent might be worth looking at